i18n/en.toml

```diff
@@ -106,6 +106,18 @@
 [connect-blogs-link-text]
   other = "View"
 
+[connect-spec-calendar-title]
+  other = "Jakarta EE Specifications Meetings Calendar"
+
+[connect-spec-calendar-body]
+  other = "LOREM IPSUM."
```

Comment:
Change "LOREM IPSUM"  to "Public calendar for all Jakarta EE specification projects.